packages/config/src/index.js

```diff
@@ -0,0 +1,86 @@
+const _ = require('lodash')
```

Comment:
@flotwig I agree, some of lodash is useful, but it would be nice to target just those functions over importing the entirety of lodash. I'm very much in favor of removing dependencies whenever we can, from both a bundle size and risk mitigation (either goofy changes, or security issues) 

i.e
```diff
-"lodash": "4.17.21"
+"lodash. camelCase": "^4.3.0",
```